src/python/bot/fuzzers/libfuzzer.py

```diff
@@ -319,6 +321,61 @@ def fuzz(self,
                                 artifact_prefix, additional_args)
 
 
+class FuchsiaQemuLibFuzzerRunner(new_process.ProcessRunner, LibFuzzerCommon):
+  """libFuzzer runner (when Fuchsia is the target platform)."""
+
+  def __init__(self, executable_path, default_args=None):
+    if (not environment.get_value('FUCHSIA_PKEY_PATH')) or (
+        not environment.get_value('FUCHSIA_PORTNUM')):
+      logs.log_error('FUCHSIA_PKEY_PATH and/or FUCHSIA_PORTNUM was not set')
+      return
+    self.ssh_root = [
+        'ssh', '-i',
+        str(environment.get_value('FUCHSIA_PKEY_PATH')), '-o',
```

Comment:
why str() ? 

as above if we exception out in the case this isn't set we should be able to assume that this is a valid string.

Reply:
environment.get_value('FUCHSIA_PORTNUM') returned an integer, rather than a string, and python (reasonably) complained, so i wound up throwing str() in front of the other one too, for safety.

i can't imagine a way that pkey_path would get int'ified, though, so backing that one out.